### PR TITLE
refactor(app): remove brackets of string attributes

### LINKED ISSUE #11211: App [Housekeeping]: replace curly bracket + single quote ({'string'}) with  double quote ("string")
In our code base, there are cases that are like `width={'1.25rem'}`. This should be `width="1.25"`.

The case we need to use `{}` in a tag(Flex, Box, StyledText Icon etc) would be a case we need to pass number to prop.
For example, zIndex and passing 0 as a number to a prop.

sample cases we need to update
```
<Flex
  right={0}

// Fixed
<Flex
  right="0"

============================

<StyledText as={'h3'}

// Fixed
<StyledText as="h3"

============================

<Icon size={'1.25rem`}

// Fixed
<Icon size="1.25rem"

============================
<Flex
  width={'100%'}

<Flex
  width="100%"

```

another sample case
https://github.com/Opentrons/opentrons/pull/11205/files#r928986620

### PR DESCRIPTION
**Pull request by MichaelMulieri**
# Overview
Removed brackets of string attributes in components.
Closes #11211.

# Changelog
- Replace instances of `value={'string prop'}` with `value="string prop"`

# Review requests
Confirm correct replacements. 

# Risk assessment
Low.

### CODE REVIEW

File: app/src/organisms/Devices/HeaterShakerWizard/AttachModule.tsx [hunk 1]
```diff
@@ -121,7 +121,7 @@ export function AttachModule(props: AttachModuleProps): JSX.Element {
                 deckDef={standardDeckDef as any}
                 viewBox={DECK_MAP_VIEWBOX}
                 deckLayerBlocklist={DECK_LAYER_BLOCKLIST}
-                id={'HeaterShakerWizard_AttachModule_deckMap'}
+                id="HeaterShakerWizard_AttachModule_deckMap"
                 data-testid={`AttachModule_empty_deck`}
```
Comment (MichaelMulieri) [inline app/src/organisms/Devices/HeaterShakerWizard/AttachModule.tsx:125]: In these changes I noticed what I think are unnecessary string interpolations.  Does that sound right?  If so I opened an issue #11243 that I would be glad to take on.

File: app/src/organisms/LabwarePositionCheck/DeckMap.tsx [hunk 2]
```diff
@@ -124,9 +124,9 @@ export const DeckMap = (props: DeckMapProps): JSX.Element | null => {
                 <rect
                   width={labwareDef.dimensions.xDimension - 2}
                   height={labwareDef.dimensions.yDimension - 2}
-                  fill={'none'}
+                  fill="none"
                   stroke={C_SELECTED_DARK}
-                  strokeWidth={'3px'}
+                  strokeWidth="3px'"
```
Comment (vabruzzo) [inline app/src/organisms/LabwarePositionCheck/DeckMap.tsx]: Extra trailing single quote here to remove

**Review by vabruzzo — CHANGES_REQUESTED** [review]
Thanks for this contribution, it looks great! There was one small issue with an extra single quote

File: app/src/organisms/Devices/HeaterShakerWizard/AttachModule.tsx (hunk 1 above)
Comment (vabruzzo) [reply app/src/organisms/Devices/HeaterShakerWizard/AttachModule.tsx:125]: this sounds good to me

**Review by koji — APPROVED** [review]
The changes look good to me ✨
Thank you for your contribution!

**Review by vabruzzo — APPROVED** [review]

**Pull request merged**